index.d.ts

```diff
@@ -32,7 +32,7 @@ type EncodingOption =
 type DefaultEncodingOption = 'utf8';
 type BufferEncodingOption = 'buffer' | null;
 
-export type CommonOptions<EncodingType extends EncodingOption = DefaultEncodingOption> = {
```

Comment:
Why remove all the default values for the generic types? This makes it easier to use the types for the common use case (where `Buffer`s are not used).

Reply:
It helps to be ensured that all variants of execa definition will have explicit type parameter. But I've forget about users who may use exported types in their apps, my bad. 

I'll restore those defaults, thx!